code/modules/integrated_electronics/subtypes/manipulation.dm

```diff
@@ -513,9 +513,12 @@
 	var/x_abs = CLAMP(T.x + target_x_rel, 0, world.maxx)
 	var/y_abs = CLAMP(T.y + target_y_rel, 0, world.maxy)
 	var/range = round(CLAMP(sqrt(target_x_rel*target_x_rel+target_y_rel*target_y_rel),0,8),1)
-
+	A.throwforce = 0
+	A.embedding = list("embed_chance" = 0)
 	A.forceMove(drop_location())
 	A.throw_at(locate(x_abs, y_abs, T.z), range, 3)
+	A.throwforce = initial(A.throwforce)
+	A.embedding = initial(A.embedding)
```

Comment:
is this a list. you can't initial lists, goof.

Reply:
since when? fuck